client/src/styles/fluent.scss

```diff
@@ -88,3 +88,12 @@ span.badge.status-fluenthidden {
   text-shadow: none;
 }
 
+.fluent__localised-field .font-icon-translatable {
+  color: #6610F2;
```

Comment:
Could we use a variable here? If you wait until #362 is merge then you can use `$fluent-color-localised`

Reply:
Sure, I am waiting patiently. ;)